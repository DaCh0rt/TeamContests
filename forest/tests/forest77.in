852 356
7751 8798
1783 205
4414 -3568
-4406 -2552
3171 -8800
-5882 -51
9678 -4211
-1550 1524
3397 3479
3953 -677
-6174 -4963
-7145 -79
7718 -8979
-383 -3634
-7551 -6395
-3123 -1630
-5226 -7810
-5553 3660
7494 1963
1057 -8714
-4400 7699
9578 -8792
-1965 4286
-6496 8089
3196 8282
5561 -1569
-4528 -574
5100 -6720
-6737 689
-2561 -1647
8466 1531
-4185 8027
-5818 -4161
9742 8180
3037 -2898
2522 -6792
8251 -844
-263 1974
-4657 2872
-6216 4844
6885 -4441
-8872 -2374
-6292 -9761
853 3224
3941 -6858
-5122 2499
-7404 2665
4479 9304
8970 5707
4834 -2415
-6120 3994
7384 7934
-5785 3613
8201 -4044
7580 -6720
6231 7863
-6998 2607
9758 -6762
2879 -1552
9686 -1248
-1735 -4299
1409 -4872
8358 -1808
-2432 -7003
-6386 -8341
562 2550
3556 -5905
-9029 3471
355 7199
1154 3184
7151 7697
8558 -943
2206 -1873
-9439 8246
8365 647
-2901 1193
7352 -9303
-7732 -4926
1681 -3175
-706 -5138
-5214 6667
4029 3662
-6203 -3056
-200 7800
-2806 9908
-7896 -7587
5959 4228
-1510 3366
5664 6305
-9227 5191
1300 5670
-9716 6348
-7374 -1275
6793 2111
1463 5949
3941 -586
-9515 -2581
811 8863
7885 -7063
5598 9772
-7823 -6079
1352 3217
-3493 -9262
-7658 8750
-1515 3949
-4855 4874
-1753 -4918
3474 -1788
8502 4227
9599 860
9726 4286
119 -8226
-3857 7501
-8229 -4165
-8558 -5778
3846 -5424
1234 -5235
2110 -4812
-6214 -8597
-5285 -5819
1367 -3713
-8266 -4971
-5697 -473
3086 -2307
-1241 -3693
-137 -4480
-4079 8385
-9275 -5313
3499 -7334
-3396 -6268
-8653 -420
8947 -2382
1470 -7870
-2649 3887
-5530 -220
3079 5430
3568 -7556
7434 -5265
1125 -4887
9108 -9431
-1322 6164
-1499 9625
-5055 7275
3356 7278
3094 1178
-411 543
7758 4834
-2255 2449
5051 8424
-6717 7664
2989 2281
-3962 2494
1024 -8271
-5403 -3090
9666 9832
-5586 4598
-1268 -4879
-7789 -6736
-6184 -6370
1579 -4444
-105 -3589
2659 -1545
5471 -8395
7611 380
2891 2898
7189 9136
-3727 1864
8383 6557
1331 -2247
6812 -1268
-3033 -836
-4120 -5999
-1382 -8800
-3950 -4758
-5250 -5594
-6659 -774
-4158 -6973
-6801 -9629
-6604 -9106
-6335 -9108
9669 4807
5520 -7620
6344 -1388
5205 -6338
-4945 3969
2717 -1811
-3614 3442
210 1208
-9547 4912
6943 7870
397 -6928
1542 3138
-1103 7970
1153 7153
-5802 -8695
1980 -4102
2269 -8301
-2056 -9555
-2866 -6755
-2779 7350
-2715 -941
2052 6805
-7025 -3118
-8303 -1484
-3608 -4656
-2144 4633
8942 -8372
7030 -1872
7111 8332
1799 3302
4204 8730
7541 6746
-9327 5214
-4383 -71
2273 6020
-3415 -4998
7021 5159
-8391 -3839
-4159 -1717
-5655 -9448
4080 691
8915 -9314
-3613 -1232
-7235 -6881
-8611 -3774
3238 -6143
-73 2988
-6806 8672
9583 1557
-5974 1646
-1073 -5367
5733 8539
-2237 -2271
9310 4476
-277 5553
-518 -1663
-8600 -6160
-7680 -6535
7191 -9562
433 -4455
-1134 1646
548 3368
5988 7575
2454 -4736
-7036 8246
-2830 -9086
-2191 -4300
-3257 -5420
7483 -5537
4404 1553
3865 -1496
909 5194
688 3768
6917 -4998
-9830 5682
-1750 -2977
-7707 -9991
-2134 -8939
-8518 4367
-4748 7221
-676 -3443
-5982 786
-1734 -9599
-7199 3706
9785 7366
-6504 1597
6753 7625
-356 -9906
-4796 -3567
-5833 7425
-9052 -4227
909 7289
-9068 4524
-1460 1989
-102 -1287
-7781 -1135
-211 -8781
-8301 4524
5018 9715
-5755 8541
6888 -936
7115 6865
-6239 2291
4609 -466
7312 -3345
6741 1262
2605 -6453
-1092 -3789
-8907 7208
5417 8330
-5758 3417
4313 -9989
5787 8461
7137 -3824
-5133 -6924
5047 3559
-6219 9265
-3379 1643
2717 2063
2357 -9437
-5363 -5595
-881 6205
-8575 4823
1518 3208
2567 624
3684 -9464
1129 8367
5039 -5566
38 7159
8498 4355
2997 -2509
-6749 -6268
4026 7996
-7013 7911
-37 1167
-7242 -4991
-9889 3355
-2291 -2143
3931 -7106
-6111 4182
-3827 4275
-8748 -8497
8241 483
6494 -3693
-6005 2830
-7732 -2524
-9671 -8175
-1915 -1828
3898 8347
6971 7722
3043 -4719
-3281 -7460
-8353 1060
-1322 7717
-2204 9637
-1059 5332
8421 -6035
-2964 -494
366 -6965
1228 8490
-4446 6131
-454 8812
-3703 4669
4579 9361
-4474 -7479
-4811 9554
5856 8418
4372 -3766
-456 8066
7321 1439
9443 -6234
-4476 -8183
5919 -4311
-9963 -4333
-8864 -5976
-9620 -2898
5108 4067
-481 -819
8122 -4554
-5333 6758
4455 -9665
-7792 -7278
-7328 -8338
-972 -824
-5319 5524
3061 3658
-9185 270
7790 -8225
-2906 5395
9217 -7818
-4045 3554
9884 665
-5648 6971
-8254 -5244
-376 3283
-1172 719
-8914 -9233
-501 -1690
-2279 1823
-3271 2343
-6314 -9371
-5360 5462
-1794 -8344
-9997 -1819
-8252 5674
-6342 7374
-2080 6097
7480 -4140
1526 -3946
9019 -7531
-7384 -1474
3684 -9494
-1527 -9028
-986 6532
4027 2175
-2889 -985
-4044 -4379
-7411 3432
-7147 5833
-3078 -7829
-4904 -630
9643 6398
2362 -5026
9746 4448
3256 -6415
-8868 6189
465 -8013
-2210 8859
9663 7877
-6618 848
-5193 -3265
5478 9084
6790 7984
8628 -1652
6619 -1238
3898 3443
-8807 -939
252 -1424
7879 -6374
1979 -3189
7638 -7572
-907 508
2763 -2532
-1955 -9540
-946 -7161
-9629 1324
551 1880
1896 2251
636 -6973
4143 6154
8550 -551
-9959 3414
-610 -9231
-3867 647
95 -9823
3551 7822
8628 6564
-5776 -512
873 -7101
-2256 859
8055 618
-8686 9037
-9549 9225
-6681 -5102
-4848 -6161
4534 5149
-726 3002
6667 -2776
-1703 -396
-4472 3730
-2118 -1697
-9372 4186
7981 -9143
1084 -6215
879 5138
9441 -9206
-2969 -9661
930 9002
6861 4526
9290 2901
1274 5632
5449 963
6010 436
7400 -7732
-867 -1839
-3863 6219
-5948 -3899
-7682 -553
-4733 -4568
9634 -6345
-9345 -2828
5866 2754
-3924 -5088
352 -3027
4120 2192
-6072 9685
-3739 -9325
3519 -1641
-2661 -9417
4766 7374
-9000 -5798
-7365 -7388
7846 2607
6330 -9052
-9593 -2209
750 2784
-1718 -1371
-6674 -4695
3422 -2851
-3320 -4755
1851 541
-1022 -7453
6787 -1079
9394 -5422
671 -2896
9329 -7719
-391 4937
-8720 -2991
2696 -7843
3894 5955
5322 5191
-7975 548
6619 -1063
1774 2326
3094 -16
-9657 -2197
-6062 857
1001 -874
8666 7750
-6620 -7838
590 -6664
8868 -2786
-5089 809
9245 -712
-1698 -554
6555 -9636
4461 -6237
-9497 -1365
-8419 5208
9223 1794
-5772 -3004
4914 4031
-5351 -3948
8697 8403
-1217 -4545
-9324 9057
-4260 7228
-2634 8266
-880 1939
2745 -5055
-3359 -6420
-4804 1380
-5426 127
5006 -2387
-8241 -7422
9276 8538
-831 7175
9292 -6305
7120 8
-5365 5316
3878 -7459
-6934 6147
945 -2437
7845 279
-4451 -9512
8202 -6638
3176 -7801
-1104 -5870
6934 -4358
-9191 2890
875 -8184
-4728 -7119
7708 -5614
1054 -8951
-9551 4582
9197 -3033
-160 -9590
-2229 -7812
-2347 1138
-30 989
6453 5920
-2903 -8609
92 9939
4401 -9502
2721 750
1828 -961
-2817 3344
-7533 5365
1157 9634
-4542 -2162
-2421 -374
6544 7351
-1472 -4442
6949 565
4860 -8345
3928 1617
3818 8170
-6263 1660
-6943 933
-7664 7538
386 2616
3599 -4087
-2887 -2963
11 -2848
5537 5684
4264 -4496
-4412 -7880
-7291 882
2489 1352
1605 -4496
-1620 -4113
-1504 6683
-417 -2530
-5011 3325
-9967 3059
6856 9177
4669 7627
9648 -3719
8986 -148
-7775 -6870
1035 -5526
-4771 6534
912 2459
2148 -6599
-5660 -4884
-1310 -3048
-9790 5780
7292 -3034
8909 5612
-1391 -2740
-1302 -8409
-1339 -1025
-4517 303
5201 -4744
4355 6963
-9822 4715
-1261 -7501
7986 -7617
-1387 4797
7029 -4084
-345 -3875
5039 9206
-9740 -5452
2650 348
8621 -4095
2235 5425
-7902 -2305
-7148 -2155
5480 -8305
2823 8773
-5860 -1306
3373 3876
9771 -7606
-7069 5035
-4473 -2651
-1662 7687
-8641 -1149
-7917 -7535
-8989 2822
-9787 -1404
8773 8162
8560 7241
7591 6576
8273 -8348
5612 5756
3859 1455
3235 4086
8378 -5569
-4451 -535
7815 -2894
5316 2484
-3984 -316
4325 -2755
-7129 -7261
6652 7874
4871 3904
8479 4559
-2808 -6422
-3351 -3884
8217 -1022
-8440 4439
7485 229
-4682 716
9192 2618
4862 -9494
-1621 2513
-8764 1503
3706 -2506
778 2741
-7952 -3330
-6526 -965
7 3701
3029 -67
5207 7026
9463 -6955
-7604 3967
4040 8147
8739 268
6918 -1609
-3139 6534
-8211 -7742
-4711 4909
-509 6623
4757 -8837
1200 -6767
3471 -5704
-5082 -6554
-4716 -7290
5552 3019
-9052 9279
3083 411
6024 7139
4906 -4262
-8545 -6034
-135 -9720
9468 -9246
9043 4115
-6601 7513
979 -5876
-3009 1206
-8782 -643
2160 8154
1337 6737
8880 -6389
-5431 -1806
-4254 -9643
-4432 8349
5609 -7452
2286 -555
9482 -4304
-3567 -4329
-9619 8631
4430 -5414
3262 5588
8499 -518
-9547 3048
5779 -7732
-3054 -8316
-5473 -8743
9994 3663
-7995 8976
6160 -1071
6750 -609
6043 9525
-6258 2853
-1427 629
2149 -1881
-3476 5068
-162 9366
8547 -6310
6582 3407
-9217 4083
-3766 -8045
-8436 -269
-7008 -9740
3525 9246
8514 -3521
-516 -2097
-3921 8820
-6758 6202
-4132 2449
-6287 6986
-2538 -4139
-5917 -8762
-9578 8599
-7256 -6993
2951 -919
8029 5231
7289 -6347
6285 6351
-5633 -7545
-1051 7543
-8201 5852
-696 -7297
-1916 9026
-4730 1945
-6076 -2863
1818 -9181
3933 2778
-1500 3602
4146 1800
-5844 -5317
-4745 4845
1043 -2081
-8156 3667
9068 -1496
-146 -7219
2513 5127
2701 -6815
-9871 -1486
-6799 9859
-6557 2751
5209 9948
6316 -3619
-1087 2653
-8120 -8693
7577 4420
931 -732
-5743 3260
7237 5977
-9766 -9981
-3277 -5062
9184 -6062
5878 -9527
-8957 -9191
86 2190
-8398 -3460
-1510 4235
8888 7111
6106 -9364
6496 -8185
6431 4153
-1906 4518
1569 6159
-3171 3784
1126 -1969
-209 -4956
-4915 -1561
-805 5564
-5844 -1262
-8205 -3351
1863 6766
-8628 -330
-6696 1228
-4373 -3829
3143 -3651
9571 -7446
5437 3667
6767 -9214
-5024 1307
1920 1914
-489 7625
-8609 342
7985 -2292
-1175 -7578
-6075 -2361
-7556 4724
-7909 -8205
9093 9259
1345 5208
-6634 190
-9872 2132
7183 -4779
-893 1270
-6528 -7692
9631 -3004
6197 1805
61 -225
-8621 8339
-1657 5780
2020 212
7513 -3856
8964 821
-9436 -6911
215 6590
5886 -624
2809 2269
-1980 -4982
-3872 7757
-543 -3115
3866 8235
3875 -4995
4997 5260
5135 -9881
-9483 -6963
-804 7855
254 -6369
-2034 -7495
293 5244
4251 -8424
-9457 -1961
9565 561
