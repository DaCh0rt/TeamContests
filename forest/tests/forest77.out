79179.89 417284409.52
